spec/reek/cli/application_spec.rb

```diff
@@ -136,4 +136,44 @@
       end
     end
   end
+
+  describe 'show configuration path' do
+    let(:app) { described_class.new ['--show-configuration-path', 'clean.rb'] }
+    let(:clean_file) do
+      <<-EOS
+        # Well documented Clean class.
+        class Clean
+          def hello(argument)
+            say argument
+          end
+        end
+      EOS
+    end
+
+    around do |example|
+      Dir.mktmpdir('/tmp') do |tmp|
+        Dir.chdir(tmp) do
+          File.write('clean.rb', clean_file)
```

Comment:
I think this spec will also work without writing a file here.

Reply:
Just passed '.' as sources and removed both clean.rb and the File.write.